---
layout: page
title: About Me
bigimg: "/img/windsurfing_picture.jpg"
---

## Breif Bio
<p align="justify">
<figure>
<div style="float: left; padding-right: 25px; padding-bottom: 25px">
	<img src="/img/bio_image.jpg" width="300" alt="" align="left">
</div>
</figure>

#Undergraduate Training
I acquired my first research training experiences during the spring of 2011 in the laboratory of Dr. Richard Veilleux (Department of Horticulture) as an undergraduate student in the Department of Biochemistry at Virginia Tech. My initial research projects involved a combination of wet lab and computational approaches to screen potential candidate genes involved with zinc and iron acquisition in potatoes. During this time, I gained experience with basic laboratory protocols such as DNA/RNA isolation, PCR, and real-time quantification. I also acquired a fascination with bioinformatic analysis, leading to the developed of computational pipelines for assessing protein structural variation utilizing massive nucleotide polymorphism data sets. Exposure to these large data sets motivated me to learn how to program, a valuable asset for analyzing contemporary genomics data.
The completion of my initial research project opened the door for further research opportunities, where in 2012, I was awarded an undergraduate research fellowship (SURF) to pursue an additional project for transposon-tagging and mutagenesis in A. thaliana. The goal of this project was to develop unique transposon-insertion lines for elucidating gene function in a newly sequenced species. I analyzed thousands of distinct transgenic A. thaliana lines generated by a meiotic-specific promoter driving the activation of the AcDs transposon-tagging system within each unique meiosis. The completion of this work resulted in the curation of a valuable resource of transposon-tagged lines for reverse-genetic engineering purposes. I presented my findings at the Summer Undergraduate Research Symposium at Virginia Tech, gaining my first taste in communicating science. These experiences afforded me a valuable framework of technical, independent thinking and communication skills, prompting me to seek additional training in a Ph.D. program. </p>

#Doctoral Training
<p align="justify">
I received my Ph.D. in the Spring of 2018 with training in the laboratory of Dr. Jiming Jiang at the University of Wisconsin-Madison (now at Michigan State University). The focus of my initial project aimed to characterize the fine-scale genetic architecture of yield in diploid potatoes. At the time, population genotyping via skim sequencing – although in its infancy – was becoming recognized for its power in studies of quantitative trait loci. As a junior PhD student armed with sequencing data from a segregating heterotic population and an insatiable appetite for literature, I realized that the potential of these data sets was not limited to linkage mapping; they could reveal meiotic crossover breakpoints at high-resolution. I was lucky to have an advisor dedicated to ensuring his student’s success and in turn, I was afforded a tremendous amount of scientific independence to pursue my own curiosities. Encouraged by the work of geneticists studying meiosis in humans, I became fascinated by the molecular framework governing recombination in plants. My interest in recombination – initially focused on the theoretical and evolutionary implications – began to take on a more foundational perspective centered on the chromatin and sequence determinants of meiotic crossovers. </p>

# Postdoctoral Training
<p align="justify">
Currently, I am an NSF NPGI postdoctoral fellow in the Schmitz Laboratory at the University of Georgia. Although the bulk of my dissertation was spent investigating the molecular and evolutionary influences of recombination, my passion is aimed at understanding the bases of chromatin architecture in transcriptional regulation. Broadly, I am interested in the origin, role and mechanism of cis-regulatory elements (CRE) in transcription and phenotypic evolution. As sessile organisms, plant genomes must contain a repertoire of programmed responses to myriad of environmental stimuli. In parallel to a fluctuating environment, plants also undergo programmed developmental changes, requiring carefully coordinated activation and silencing of specific transcriptional programs. The control of transcriptional programs for development and biotic/abiotic responses are likely established and modulated by CREs involved in complex chromatin looping architectures. Past studies in early eukaryotes implicated enhancer CREs (gene-distal transcriptional regulators) as a metazoan lineage-specific innovation. A growing body of evidence, including genome-wide and functional assays in maize, rice, potato and Arabidopsis now highlight the wide-ranging prevalence of gene-distal CREs in plant genomes, and implicate a central function in transcriptional regulation. Building on these recent findings, I am focused on elucidating how variation in cis-regulatory elements, chromatin structure and patterns of transcription influence development, environmental responses, and phenotypic evolution in plants. To address these questions, I am constructing molecular and computational tools for generating and analyzing chromatin and transcriptomic data in plants from a diverse set of tissues, genotypes and species. </p>
