---
layout: page
title: Publications
bigimg: "/img/publication_image.png"
---

Publications and preprints tracked by Google Scholar can be found here: [Google Scholar link](https://scholar.google.com/citations?user=_bYW4UkAAAAJ&hl=en)

### 2022
25\.    [`New Machine learning reveals conserved chromatin patterns determining meiotic recombination sites in plants`](https://www.biorxiv.org/content/10.1101/2022.07.11.499557v1.abstract)\
Minghui Wang, Shay Shilo, Adele Zhou, Mateusz Zelkowski, Mischa A. Olson, Ido Azuri, Nurit Shoshani-Hechel, Cathy Melamed-Bessudo, Alexandre P. Marand, Jiming Jiang, James C. Schnable, Charles J. Underwood, Ian R. Henderson, Qi Sun, Jaroslaw Pillardy, Penny M.A. Kianian, Shahryar F. Kianian, Changbin Chen, Avraham A. Levy, Wojciech P. Pawlowski. (2022)\
**_bioRxiv_**

<html>
  <style>
    section {
        background: white;
        color: black;
        border-radius: 1em;
        padding: 1em;
        left: 50% }
    inner {
        display: inline-block;
        display: flex;
        align-items: center;
        justify-content: center }
  </style>
  <section>
    <div id="inner">
        <span style="float:left";
          class="__dimensions_badge_embed__"
          data-doi="10.1101/2022.07.11.499557"
          data-hide-zero-citations="true"
          data-legend="always">
        </span>
      <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
        <div style="float:right";
          data-link-target="_blank"
          data-badge-details="right"
          data-badge-type="medium-donut"
          data-doi="10.1101/2022.07.11.499557"   
          data-condensed="true"
          data-hide-no-mentions="true"
          class="altmetric-embed">
        </div>
      <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
    </div>
    <div id="inner">
      <script type="text/javascript" src="//cdn.plu.mx/widget-summary.js"></script>
        <a href="https://plu.mx/plum/a/?doi=10.1101/2022.07.11.499557"
          data-orientation="horizontal"
          class="plumx-summary"
          data-site="plum"
          data-hide-when-empty="true">
        </a>
    </div>
  </section>
</html>


24\.    [`Modeling chromatin state from sequence across angiosperms using recurrent convolutional neural networks`](https://www.biorxiv.org/content/10.1101/2021.11.11.468292v1.abstract)\
Travis Wrightsman,  Alexandre P. Marand,  Peter A. Crisp,  Nathan M. Springer,  Edward S. Buckler. (2022)\
**_bioRxiv_**


23\.    [`A combinatorial indexing strategy for low cost epigenomic profiling of plant single cells.`](https://doi.org/10.1016/j.xplc.2022.100308)\
Alexandre P. Marand, Xiaoyu Tu, Robert J. Schmitz, Silin Zhong. (2022)\
**_Plant Communications_**



22\.    [`Single-cell analysis of cis-regulatory elements.`](https://doi.org/10.1016/j.pbi.2021.102094)\
Alexandre P. Marand, Robert J. Schmitz. (2022)\
**_Current Opinion in Plant Biology_**



### 2021
21\.    [`Quality control and evaluation of plant epigenomics data.`](https://doi.org/10.1093/plcell/koab255)\
Robert J. Schmitz, Alexandre P. Marand, Xuan Zhang, Rebecca A. Mosher, Franziska Turck, Xuemei Chen, Michael J. Axtell, Xuehua Zhong, Siobhan M. Brady, Molly Megraw, Blake C. Myers. (2021)\
**_The Plant Cell_**



20\.    [`Vision, challenges, and opportunities for a Plant Cell Atlas.`](https://doi.org/10.7554/eLife.66877)\
Plant Cell Atlas Consortium, Suryatapa Ghosh Jha, Alexander T. Borowsky, Benjamin J. Cole, Noah Fahlgren, Andrew Farmer, Shao-shan Carol Huang, Purva Karia, Marc Libault, Nicholas J. Provart, Selena L. Rice, Maite Saura-Sanchez, Pinky Agarwal, Amir H. Ahkami, Christopher R. Anderton, Steven P. Briggs, Jennifer A.N. Brophy, Peter Denolf, Luigi F. Di Costanzo, Moises Exposito-Alonso, Stefania Giacomello, Fabio Gomez-Cano, Kerstin Kaufmann, Dae Kwan Ko, Sagar Kumar, Andrey V. Malkovskiy, Naomi Nakayama, Toshihiro Obata, Marisa S. Otegui, Gergo Palfalvi, Elsa H. Quezada-Rodríguez, Rajveer Singh, R. Glen Uhrig, Jamie Waese, Klaas Van Wijk, R. Clay Wright, David W. Ehrhardt, Kenneth D. Birnbaum, Seung Y. Rhee. (2021)\
**_eLife_**



19\.    [`Profiling single-cell chromatin accessibility in plants.`](https://doi.org/10.1016/j.xpro.2021.100737)\
Alexandre P. Marand, Xuan Zhang, Julie Nelson, Pedro Augusto Braga dos Reis, Robert J. Schmitz. (2021)\
**_STAR Protocols_**



18\.    [`De novo assembly, annotation, and comparative analysis of 26 diverse maize genomes.`](https://doi.org/10.1126/science.abg5289)\
Matthew B. Hufford, Arun S. Seetharam, Margaret R. Woodhouse, Kapeel M. Chougule, Shujun Ou, Jianing Liu, William A. Ricci, Tingting Guo, Andrew Olson, Yinjie Qiu, Rafael Della Coletta, Silas Tittes, Asher I. Hudson, Alexandre P. Marand, Sharon Wei, Zhenyuan Lu, Bo Wang, Marcela K. Tello-Ruiz, Rebecca D. Piri, Na Wang, Dong won Kim, Yibing Zeng, Christine H. O'Connor, Xianran Li, Amanda M. Gilbert, Erin Baggs, Ksenia V. Krasileva, John L. Portwood II, Ethalinda K.S. Cannon, Carson M. Andorf, Nancy Manchanda, Samantha J. Snodgrass, David E. Hufnagel, Qiuhan Jiang, Sarah Pedersen, Michael L. Syring, David A. Kudrna, Victor Llaca, Kevin Fengler, Robert J. Schmitz, Jeffrey Ross-Ibarra, Jianming Yu, Jonathan I. Gent, Candice N. Hirsch, Doreen Ware, R. Kelly Dawe. (2021)\
**_Science_**



17\.    [`Leveraging histone modifications to improve genome annotations.`](https://doi.org/10.1093/g3journal/jkab263)\
John P. Mendieta, Alexandre P. Marand, William A. Ricci, Xuan Zhang, Robert J. Schmitz. (2021)\
**_G3 Genes\|Genomes\|Genetics_**




16\.    [`Stability of DNA methylation and chromatin accessibility in structurally diverse maize genomes.`](https://doi.org/10.1093/g3journal/jkab190)\
Jaclyn M. Noshay, Zhikai Liang, Peng Zhou, Peter A. Crisp, Alexandre P. Marand, Candice N. Hirsch, Robert J. Schmitz, Nathan M. Springer. (2021)\
**_G3 Genes\|Genomes\|Genetics_**



15\.    [`A cis-regulatory atlas in maize at single-cell resolution.`](https://doi.org/10.1016/j.cell.2021.04.014)\
Alexandre P. Marand, Zongliang Chen, Andrea Gallavotti, Robert J. Schmitz. (2021)\
**_Cell_**


Coverage:\
[PNAS news](https://www.pnas.org/post/journal-club/atlas-identifies-genome-regions-that-regulate-plant-cell-identity)\
[Plantae](https://plantae.org/a-cis-regulatory-atlas-in-maize-at-single-cell-resolution-biorxiv/)\
[UGA Research](https://www.genetics.uga.edu/news/stories/2021/dr-marand-leads-project-applying-single-cell-sequencing-technology-create)


14\.    [`Chromatin accessibility profiling methods.`](https://doi.org/10.1038/s43586-020-00008-9)\
Liesbeth Minnoye, Georgia K. Marinov, Thomas Krausgruber, Lixia Pan, Alexandre P. Marand, Stefano Secchia, William J. Greenleaf, Eillen E. M. Furlong, Keigi Zhao, Robert J. Schmitz, Christoph Bock, Stein Aerts. (2021)\
**_Nature Reviews Methods Primers_**



### 2020
13\.    [`Assessing the regulatory potential of transposable elements using chromatin accessibility profiles of maize transposons.`](https://doi.org/10.1093/genetics/iyaa003)\
Jaclyn M. Noshay, Alexandre P. Marand, Sarah N. Anderson, Peng Zhou, Maria Katherine Mejia Guerra, Zefu Lu, Christine O’Connor, Peter A. Crisp, Candice N. Hirsch, Robert J. Schmitz, Nathan M. Springer. (2020)\
**_Genetics_**



12\.    [`Stable unmethylated DNA demarcates expressed genes and their cis-regulatory space in plant genomes.`](https://doi.org/10.1073/pnas.2010250117)\
Peter A. Crisp, Alexandre P. Marand, Jaclyn M. Noshay, Peng Zhou, Zefu Lu, Robert J. Schmitz, Nathan M. Springer. (2020)\
**_Proceedings of the National Academy of Sciences_**



11\.    [`Identification of a novel base J binding protein complex involved in RNA polymerase II transcription termination in trypanosomes.`](https://doi.org/10.1371/journal.pgen.1008390)\
Rudo Kieft, Yang Zhang, Alexandre P. Marand, Jose Dagoberto Moran, Robert Bridger, Lance Wells, Robert J. Schmitz, Robert Sabatini. (2020)\
**_PLOS Genetics_**



### 2019
10\.    [`The prevalence, evolution and chromatin signatures of plant regulatory elements.`](https://doi.org/10.1038/s41477-019-0548-z)\
Zefu Lu, Alexandre P. Marand, William A. Ricci, Christina L. Ethridge, Xiaoyu Zhang, Robert J. Schmitz. (2019)\
**_Nature Plants_**



9\.    [`Widespread long-range cis-regulatory elements in the maize genome.`](https://doi.org/10.1038/s41477-019-0547-0)\
William A. Ricci, Zefu Lu, Lexiang Ji, Alexandre P. Marand, Christina L. Ethridge, Nathalie G. Murphy, Jaclyn M. Noshay, Mary Galli, María Katherine Mejía-Guerra, Maria Colomé-Tatché, Frank Johannes, M. Jordan Rowley, Victor G. Corces, Jixian Zhai, Michael J. Scanlon, Edward S. Buckler, Andrea Gallavotti, Nathan M. Springer, Robert J. Schmitz, Xiaoyu Zhang. (2019)\
**_Nature Plants_**



8\.    [`Cold stress induces enhanced chromatin accessibility and bivalent histone modifications H3K4me3 and H3K27me3 of active genes in potato.`](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1731-2)\
Z Zeng, W Zhang, Alexandre P. Marand, B Zhu, CR Buell, J Jiang. (2019)\
**_Genome Biology_**



7\.    [`Residual Heterozygosity and Epistatic Interactions Underlie the Complex Genetic Architecture of Yield in Diploid Potato.`](https://www.genetics.org/content/212/1/317.abstract)\
Alexandre P. Marand, SH Jansky, JL Gage, AJ Hamernik, N de Leon, J Jiang. (2019)\
**_Genetics_**


[Article Highlights](https://www.genetics.org/content/212/1/NP)


6\.    [`Historical Meiotic Crossover Hotspots Fueled Patterns of Evolutionary Divergence in Rice.`](http://www.plantcell.org/content/31/3/645.abstract)\
Alexandre P. Marand, H Zhao, W Zhang, Z Zeng, C Fang, J Jiang. (2019)\
**_The Plant Cell_**


[F1000 recommended article](https://f1000.com/prime/734986434)


### 2018
5\.    [`Genome sequence of M6, a diploid inbred clone of the high-glycoalkaloid-producing tuber-bearing potato species Solanum chacoense, reveals residual heterozygosity.`](https://doi.org/10.1111/tpj.13857)\
CP Leisner, JP Hamilton, ..., Alexandre P. Marand, ..., SH Jansky, CR Buell. (2018)\
**_The Plant Journal_**


4\.    [`Proliferation of regulatory DNA elements derived from transposable elements in the maize genome.`](http://www.plantphysiol.org/content/176/4/2789.abstract)\
H Zhao, W Zhang, L Chen, Alexandre P. Marand, Y Wu, J Jiang. (2018)\
**_Plant Physiology_**



### 2017
3\.    [`Meiotic crossovers are associated with open chromatin and enriched with Stowaway transposons in potato.`](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1326-8)\
Alexandre P. Marand, SH Jansky, H Zhao, CP Leisner, X Zhu, Z Zeng, E Crisovan, L Newton, AJ Hamernik, RE Veilleux, CR Buell, J Jiang. (2017)\
**_Genome Biology_**



2\.    [`Towards genome-wide prediction and characterization of enhancers in plants.`](https://www.sciencedirect.com/science/article/pii/S1874939916301274)\
Alexandre P. Marand, T Zhang, B Zhu, J Jiang. (2017)\
**_BBA Gene Regulatory Mechanisms_**



### 2016
1\.    [`PlantDHS: a database for DNase I hypersensitive sites in plants.`](https://academic.oup.com/nar/article-abstract/44/D1/D1148/2503132)\
T Zhang, Alexandre P. Marand, J Jiang. (2016)\
**_Nucleic Acids Research_**
