---
layout: page
title: Research
bigimg: "/img/Figure_1.png"
---

# Motivation

<p align="justify">
The long term goal of the lab is to understand how the genome of a single cell encodes the development of an entire multicellular organism composed of millions, billions, and even trillions of cells. Cellular diversity within and between organisms is largely attributed to differences in transcriptional regulation, with cis-regulatory elements - sequences that control when and where a gene is transcribed - playing a central role. To tackle this question, the lab is currently focused on three major objectives:</p>

1. Identify the comprehensive catalog of *cis*-regulatory elements and their associated *trans*-factors underlying diverse cell identities and fate transitions in plant genomes. Leveraging recent advances in single-cell profiling methods, the lab primarily focuses these efforts in *Zea mays* and *Arabidopsis thaliana* and aims to expand to other model systems.

2. The lab is invested in developing new techniques (experimental and computational) for quantitatively assaying non-coding regulatory element function. Efforts to chart the locations and functions of silencers, chromatin organization elements, and other cis-regulatory types have remained relatively unexplored compared to enhancers and promoters, including in animal and invertebrate systems. Determining how combinatorial interactions between distinct regulatory elements results in a specific quantitative activity is a major objective of the lab.

3. Understanding how (epi)genetic and (epi)genomic variation affects cis-regulatory activity and transcription regulation at the molecular and mechanistic levels. Coupling standing genetic variation and mutagenesis approaches with high-throughput sequencing and reporter assays at the single-cell level facilitates characterization of novel molecular mechanisms underlying plasticity in cellular function, identity, and development.


## Single-cell genomics for decoding development

<p align="justify">
A major goal of developmental biology is elucidating how morphological and developmental variation arises within a species. Accumulating evidence from Genome-wide Association Studies implicates genetic variation within cis-regulatory elements as a significant source affecting phenotypic traits. Current paradigms suggest that regulatory variation may affect chromatin structure and transcription factor binding that result in altered patterns of transcription that encourage phenotypic evolution. Results from the lab indicate significant enrichment of phenotype-associated genetic variants within cell-type-specific cis-regulatory elements, suggesting that trait-associated variation are affecting transcriptional regulation in specific cellular contexts.
</p>

<p align="justify">
<figure>
<div style="float: left; padding-right: 25px; padding-bottom: 25px">
	<img src="/img/GWAS_enrichment_celltypespecific.png" width="450" alt="" align="left">
</div>
</figure>
</p>

<p align="justify">
<figure>
<div style="float: left; padding-right: 25px; padding-bottom: 25px">
	<img src="/img/282_UMAP.png" width="250" alt="" align="left">
</div>
</figure>
</p>

<p align="justify">
To better chart the genetic determinants of regulatory function, we are using single-cell genomics to profile chromatin accessibility in diverse genetic backgrounds in maize. This project aims to identify genetic variants affecting levels chromatin accessibility in diverse cellular contexts, their effects on transcript abundance, and the consequences on cellular and organism-scale phenotypes. A comprehensive map of molecular effects will enable future genome engineering efforts to predictably fine-tune important traits.
</p>

<p align="justify">
<figure>
<div style="float: left; padding-right: 25px; padding-bottom: 25px">
	<img src="/img/282_chromatin_variation.png" width="350" alt="" align="left">
</div>
</figure>
</p>
